proof/invariant-abstract/ARM/ArchInvariants_AI.thy

```diff
@@ -747,6 +747,12 @@ definition
 where
   "hyp_live ko \<equiv> False"
 
+definition arch_tcb_live :: "arch_tcb \<Rightarrow> bool" where
+  "arch_tcb_live arch_tcb \<equiv> False"
```

Comment:
usually we want a comment for these, e.g. "No FPU on this platform" or whatever is the reason